Resources/Locale/en-US/flavors/flavor-profiles.ftl

```diff
@@ -112,7 +112,7 @@ flavor-complex-corn = like corn
 flavor-complex-banana = like bananas
 flavor-complex-apple = like apples
 flavor-complex-cotton = like cottons
-flavor-complex-bungo = like bungo
```

Comment:
:(

Reply:
Im on mobile right now cant tell what this ones about